migrations/000005_new_tables.up.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TYPE access AS ENUM ('user', 'admin');
+
+CREATE TABLE IF NOT EXISTS public.userstable
+(
+    user_id uuid PRIMARY KEY NOT NULL,
+    name text   NOT NULL,
+    user_access access DEFAULT 'user'
+);
+
+CREATE TYPE order_status AS ENUM ('accepting_items', 'canceled', 'waiting_payment', 'paid');
+
+CREATE TABLE IF NOT EXISTS public.orderstable
+(
+order_id		uuid PRIMARY KEY NOT NULL,
+purchaser_id 	uuid,	
+order_status	order_status DEFAULT 'accepting_items',
+created_at 		timestamp with time zone DEFAULT now(),
+updated_at 	timestamp with time zone DEFAULT now()
```

Comment:
What a awkward formatting. If you use spaces instead of tabs you wont have this issue.
Just don't get sucked in by the [holy wars](https://www.google.com/search?q=tabs+vs+spaces&sca_esv=297f8d6bee6ef7a9&source=hp&ei=MY3UZd_rJa7Z5OUP4pum4Aw&iflsig=ANes7DEAAAAAZdSbQfSrc1CoiH8SrSyNaZuq6m8s0szW&udm=&ved=0ahUKEwifyMm76LmEAxWuLLkGHeKNCcwQ4dUDCBU&uact=5&oq=tabs+vs+spaces&gs_lp=Egdnd3Mtd2l6Ig50YWJzIHZzIHNwYWNlczIFEAAYgAQyBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB5I0xlQAFi4GHABeACQAQCYAX6gAbUNqgEEMC4xNbgBA8gBAPgBAcICCxAAGIAEGLEDGIMBwgIREC4YgAQYsQMYgwEYxwEY0QPCAggQLhiABBixA8ICCBAAGIAEGLEDwgIIEC4YsQMYgATCAgcQABiABBgKwgILEC4YgAQYxwEYrwHCAgUQLhiABMICChAAGIAEGAoYsQPCAggQABgWGB4YCg&sclient=gws-wiz).